pgxn/neon/file_cache.c

```diff
@@ -186,8 +210,18 @@ lfc_shmem_startup(void)
 		lfc_ctl->used = 0;
 		dlist_init(&lfc_ctl->lru);
 
-		/* Remove file cache on restart */
-		(void)unlink(lfc_path);
+		/* Recreate file cache on restart */
+		fd = BasicOpenFile(lfc_path, O_RDWR|O_CREAT|O_TRUNC);
```

Comment:
Why not throw in O_EXCL there for good measure?

Reply:
Is there warranty that file is removed after restart?
Whjo and how should enforce it?